public/app/features/dashboard-scene/scene/layout-default/RowRepeaterBehavior.ts

```diff
@@ -196,15 +194,22 @@ export class RowRepeaterBehavior extends SceneObjectBase<RowRepeaterBehaviorStat
 
         const cloneItemKey = joinCloneKeys(rowCloneKey, getLastKeyFromClone(sourceItem.state.key!));
         const cloneItemY = sourceItemY + (rowContentHeight + 1) * rowIndex;
-
-        const cloneItem = sourceItem.clone({
+        const cloneItem =
```

Comment:
i realize this is a backport, but would be good to add a comment as to why this is necessary